docker-run-generate-outputs.sh

```diff
@@ -22,6 +22,9 @@ while [[ $# -gt 0 ]]; do
     esac
 done
 
+# Change to true when generating analysis files
+ANALYSIS_FILE_MODE=true
```

Comment:
Duplicating and hard-coding this deep inside the code isn't great. But I do see why you needed it.
Not sure what the best thing to do instead is. Possible options:
 - Have this be a command line flag only rather than live in configuration json
 - Change this script to only copy out analysis files if they exist (rather than if this flag is true), and possibly warn if they don't.

Feel free to bring up for discussion in team_data_tech.

Reply:
Thanks @as2388  moved to cmd line argument with two options true/false.